### PR TITLE
Project: Split BUILD file into several. Part 1/3

### PR DESCRIPTION
**Pull request by russellhancox**
The tests fail in this PR because the rules need updating. I'll fix them in a follow-up PR.

### CODE REVIEW

File: Source/SantaGUI/BUILD [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    xibs = [
+        "Resources/AboutWindow.xib",
+        "Resources/MessageWindow.xib",
+    ],
+    deps = [
+        "//Source/common:SNTBlockMessage",
+        "//Source/common:SNTConfigurator",
+        "//Source/common:SNTXPCControlInterface",
+        "//Source/common:SNTXPCNotifierInterface",
+        "@MOLCodesignChecker//:MOLCodesignChecker",
```
Comment (tburgin) [inline Source/SantaGUI/BUILD]: nit: @MOLCodesignChecker
Maybe just run buildifier on all the BUILD files?

Comment (russellhancox) [reply Source/SantaGUI/BUILD]: Done

**Comment by russellhancox** [comment]
I was in two minds about the glob() thing - I like being quite specific rather than relying on glob but the thing that pushed me into avoiding it is that I'm moving the test source files to live with the libs they're testing and keeping the exclusion list to exclude those is annoying.

File: Tests/LogicTests/BUILD [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        "DiskArbitration",
+        "IOKit",
+        "SecurityInterface",
+    ],
+    structured_resources = glob([
+        "Tests/LogicTests/Resources/BundleExample.app/**",
+        "Tests/LogicTests/Resources/DirectoryBundle/**",
+    ]),
+    deps = [
+        "@FMDB//:fmdb",
```
Comment (tburgin) [inline Tests/LogicTests/BUILD]: @FMDB

**Comment by tburgin** [comment]
Ack, makes sense.

**Pull request merged**